docs/scripts/config.js

```diff
@@ -87,7 +87,8 @@ const enhance = (schema, parents = []) => (item) => {
     })
   }
 
-  if (!hasChildren) {
+  const showEnvVarBlock = pathOr('', [...path, 'showEnvVarBlock'], schema)
+  if (!hasChildren || showEnvVarBlock) {
```

Comment:
I like this, but can you add it here: https://github.com/ory/docusaurus-template/blob/master/scripts/config.js ? 
Sorry for all these separate PRs, but it will then be available in all projects :wink:

Reply:
Maybe we can call the key `showEnvVarRawJSON` and also add the text you specified in the description?

Reply:
Made the name a bit clearer, though I figured `showEnvVarBlockForObject` would be a bit clearer because it's about the type of the config option rather than the content of the value. I also changed the logic to automatically show the text if it's an object and this field is set. (Ported this change back from my PR in docusaurus. Not sure if that's necessary but I figured I'd just keep them in sync.

Reply:
I will ping you once everything propagated to master

Reply:
Don't expect it today though, I will go to bed soon :sweat_smile: 
When CI passed in https://github.com/ory/kratos/pull/836 and it is merged, we can merge yours. I will update your branch in the morning, as you are probably behind us a view hours.

Reply:
Thanks, that's no problem!

Reply:
Ok merged, if you want you can rebase. Also, can you maybe run the config docs gen script and commit the updated file?

Reply:
There seem to be quite a few other changes as well. Should I go ahead and commit them here or do you typically merge a branch in and then regenerate on `master`?

Reply:
I just merged it and ran `cd docs; npm run gen` so we can see your change in action :wink: